prow/cmd/deck/job_history_test.go

```diff
@@ -44,63 +44,63 @@ func TestJobHistURL(t *testing.T) {
 			address:         "http://www.example.com/job-history/foo-bucket/logs/bar-e2e",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/foo-bucket/logs/bar-e2e?buildId=",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/foo-bucket/logs/bar-e2e?buildId=123456789123456789",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              123456789123456789,
 		},
 		{
 			address:         "http://www.example.com/job-history/gs/foo-bucket/logs/bar-e2e",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/gs/foo-bucket/logs/bar-e2e?buildId=",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/gs/foo-bucket/logs/bar-e2e?buildId=123456789123456789",
 			bktName:         "foo-bucket",
 			storageProvider: providers.GS,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              123456789123456789,
 		},
 		{
 			address:         "http://www.example.com/job-history/s3/foo-bucket/logs/bar-e2e",
 			bktName:         "foo-bucket",
 			storageProvider: providers.S3,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/s3/foo-bucket/logs/bar-e2e?buildId=",
 			bktName:         "foo-bucket",
 			storageProvider: providers.S3,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
 			id:              emptyID,
 		},
 		{
 			address:         "http://www.example.com/job-history/s3/foo-bucket/logs/bar-e2e?buildId=123456789123456789",
 			bktName:         "foo-bucket",
 			storageProvider: providers.S3,
-			root:            "logs/bar-e2e",
+			root:            "logs/bar-e2e/",
```

Comment:
Should we include a test where the root doesn't have `/` to cover your added `/` case?

Reply:
Actually every test case checks that, the root is extracted from the `address` property, if it won't work as expected the result would be a root with no trailing `/`.

Reply:
haha got it thanks